c/serializer/samples/remote_monitoring/windowsce/remote_monitoring.sln

```diff
@@ -0,0 +1,22 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 2013
```

Comment:
Support for Windows Embedded Compact on VS2015 has been released just a couple of weeks ago, I suppose that most of the customers are still using VS2013. And (bad!) an SDK generated with VS2013 won't work on VS2015, so people that are not generating their own SDK will have to wait until OEMs release an updated SDK. I think that at the moment keeping VS2013 compatibility can be a good idea.